components/list/list-item-mixin.js

```diff
@@ -343,9 +343,9 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				transition: border-color 400ms linear;
 			}
 			:host([_highlighting]) [slot="outside-control-container"],
-			:host(:not([disabled]):not([skeleton])[_hovering][_highlighting]) [slot="outside-control-container"],
-			:host(:not([disabled]):not([skeleton])[_focusing][_highlighting]) [slot="outside-control-container"],
-			:host(:not([disabled]):not([skeleton])[selected][_highlighting]) [slot="outside-control-container"] {
+			:host(:not([disabled]):not([selection-disabled]):not([skeleton])[_hovering][_highlighting]) [slot="outside-control-container"],
+			:host(:not([disabled]):not([selection-disabled]):not([skeleton])[_focusing][_highlighting]) [slot="outside-control-container"],
+			:host(:not([disabled]):not([selection-disabled]):not([skeleton])[selected][_highlighting]) [slot="outside-control-container"] {
```

Comment:
The disabled property is also used for some styling for [list-item-link-mixin.js](https://github.com/BrightspaceUI/core/blob/main/components/list/list-item-link-mixin.js) and [list-item-button-mixin.js](https://github.com/BrightspaceUI/core/blob/main/components/list/list-item-button-mixin.js).

Reply:
> The disabled property is also used for some styling for [list-item-link-mixin.js](https://github.com/BrightspaceUI/core/blob/main/components/list/list-item-link-mixin.js) and [list-item-button-mixin.js](https://github.com/BrightspaceUI/core/blob/main/components/list/list-item-button-mixin.js).

For links, the disabled property seems to refer to different things than `selection-disabled` so I left them there just in case consumers still exist. It disables glow/ring when the primary action is disabled(I'm not sure if disabling links should exist, just remove the href-property).
For buttons, I think it will be a separate PR that also tackles [this thread](https://github.com/BrightspaceUI/core/pull/3194)

Reply:
> I'm not sure if disabling links should exist, just remove the href-property

Disabled links is not a thing. To "disable" a link, the consumer simply does not specify an href/action-href, and no link is rendered.

Reply:
The list-item-links and list-item-buttons inherit from the `ListItemMixin`, so your changes to the `disabled` property will also be reflected in places where those mixins are used.